daal4py/sklearn/monkeypatch/tests/test_patching.py

```diff
@@ -0,0 +1,88 @@
+#===============================================================================
+# Copyright 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+import re
+import subprocess
+import sys
+import os
+import pathlib
+import pytest
+from models_info import TO_SKIP
+
+
+def get_method(s):
+    return s.split('.')[3].split(':')[0]
```

Comment:
What is 3? ) Name the constants so that the reader of the code can understand what this is about.